topics:
  odom_topic: "/lidar_slam/odom"
  ctrl_topic: "/cmd_vel"
  cmd_topic: "/planning/pos_cmd"

position_gain:
  x: 1.0
  y: 1.0
velocity_gain:
  x: 1.0
  y: 1.0
attitude_gain:
  yaw: 1.0
  yaw_dot: 1.0